development.html

```diff
@@ -3,7 +3,7 @@
 	<title>Running Dungeons of Kathallion</title>
 	<body>
 	  <div class="header">
-	    <a href="index.html">Home</a> | <a href="running.html">Running Dungeons of Kathallion</a> | <span class="selected">Development</span>
+	    <a href="index.html">Home</a> | <a href="running.html">Running Dungeons of Kathallion</a> | <span class="selected">Development</span> | <a href="bane-of-wargs">Bane Of Wargs</a>
```

Comment:
```suggestion
	    <a href="index.html">Home</a> | <a href="running.html">Running Dungeons of Kathallion</a> | <span class="selected">Development</span> | <a href="bane-of-wargs.html">Bane Of Wargs</a>
```
Doesn't seem to matter, but I would like  consistency with other links.